README.md

```diff
@@ -32,6 +40,21 @@ QUICKINSTALL:
 
 After building go to the `examples` directory and have a look at the python scripts.
 
+DOCUMENTATION:
+==============
+
+Documentation for the developer version of ESPResSo++ is at:
+
+http://espressopp.github.io
+
+Documentation for release versions from v1.9.4.1 onward is at:
```

Comment:
`v1.9.4.1` was more meant as a test release for the infrastructure hence I wanted to remove it later, but I am happy to keep it.